### DISCUSSION TITLE
下载的excel有问题：部分内容有问题，是否让我们尽量尝试恢复

### DISCUSSION THREAD

**Discussion by EmirMa** [exceljs/exceljs#1902] [Q&A]
请帮我检查这个问题，地址：https://ask.csdn.net/questions/7563313
@ValerioSevilla @nicoladefranceschi @skumarnk2 @priidikvaikla @piglovesyou

**Comment by EmirMa** [comment]
使用MS-Office2016打开报这个错误；
使用WPS打开OK。

**Reply by skypesky** [reply]
@EmirMa 使用MS-Office2016打开报这个错误 => 打开文件的时候会有报错的话,你可以把这个修复记录的 `xml` 文件提供一下看看
https://img-mid.csdnimg.cn/release/static/image/mid/ask/31855989563611.png?x-oss-process=image/auto-orient,1/resize,w_320,m_lfit

**Comment by EmirMa** [comment]
xml：
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><logFileName>error243720_01.xml</logFileName><summary>在文件“C:\Users\dell\Downloads\NPS和满意度报表-2021-11-13-08-31-38.xlsx”中检测到错误</summary><additionalInfo><info>Excel 已完成文件级验证和修复。此工作簿的某些部分可能已被修复或丢弃。</info></additionalInfo></recoveryLog>


你好，大神，下载的excel如附件，感谢。

链接: https://pan.baidu.com/s/1sQ6UStRtDLvDC8oVScmtVg 
提取码: t88g 









--

Best Regards！







At 2021-11-12 15:58:50, "skypesky" ***@***.***> wrote:

@EmirMa 使用MS-Office2016打开报这个错误 => 打开文件的时候会有报错的话,你可以把这个修复记录的 xml 文件提供一下
https://img-mid.csdnimg.cn/release/static/image/mid/ask/31855989563611.png?x-oss-process=image/auto-orient,1/resize,w_320,m_lfit

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by skypesky** [reply]
@EmirMa 
好像还有一个error243720_01.xml之类的文件,这个文件的内容是什么呢?
目前从你发的问的这个文件的日志看不到任何修复的信息,因此不知道它修复了什么.
- tips: xml文件中提到了 <logFileName>error243720_01.xml</logFileName>

**Comment by EmirMa** [comment]
你好，每次打开excel都会产生这样的xml，但文件 名字随机生成的。




每次打开都会产生这个文件：




--

Best Regards！

Ext: +86-760-23382382-6118

Wistron InfoComm (Zhongshan) Corporation

Torch High-tech Industrial Development Zone, Zhongshan, Guangdong, China, 528437

Tencent Account:283979007




在 2021-11-21 19:09:57，"skypesky" ***@***.***> 写道：

@EmirMa 好像还有一个error243720_01.xml之类的文件,目前从你发的问的这个文件的日志看不到任何修复的信息,也不知道它修复了是什么滴.

tips: xml文件中提到了 error243720_01.xml

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by skypesky** [reply]
@EmirMa 文件名虽然随机,但是里面有内容应该可以告诉我们修复了什么东西.只有知道Excel修复了什么东西,我们才有机会解决这个问题哦!

**Reply by EmirMa** [reply]
xml里面的内容如下：
`<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><logFileName>error243720_01.xml</logFileName><summary>在文件“C:\Users\dell\Downloads\NPS和满意度报表-2021-11-13-08-31-38.xlsx”中检测到错误</summary><additionalInfo><info>Excel 已完成文件级验证和修复。此工作簿的某些部分可能已被修复或丢弃。</info></additionalInfo></recoveryLog>`

**Comment by skypesky** [comment]
我仔细看了一下,根据这些文件的内容看不出来到底是哪里错了.文件没指出到底修复了哪里.

**Comment by Zhang-haohua** [comment]
同样遇到这问题，希望有解决方案

**Reply by Zhang-haohua** [reply]
当我的子表名称以数字开头或者含空格的时候，就会出现上面的报错

**Reply by skypesky** [reply]
@Zhang-haohua 你有测试的代码可以复现这个错误吗?
